README.md

```diff
@@ -144,12 +144,14 @@ robot model is loaded. For example, the following XML will load the default plug
 
 #### Set up controllers
 
-Use the tag `<parameters>` inside `<plugin>` to set the YAML file with the controller configuration.
+Use the tag `<parameters>` inside `<plugin>` to set the YAML file with the controller configuration
+and use the tag `<controller_manager_node_name>` to set the controller manager node name.
 
 ```xml
 <gazebo>
   <plugin name="ignition_ros2_control" filename="libignition_ros2_control.so">
     <parameters>$(find ignition_ros2_control_demos)/config/cartpole_controller.yaml</parameters>
+    <controller_manager_node_name>controller_manager</controller_manager_node_name>
```

Comment:
Could we rather set a “prefix” which will be then used before both node names, plugin node and CM node? This is then also compatible with most robot descriptions, which have an option to set robot's prefix in multi robot systems.

The idea here is to enable to run multiple Plugins/CMs in Ignition and enable complex multi-robot simulations. What do you think?